src/cclib/parser/qchemparser.py

```diff
@@ -481,6 +483,12 @@ def extract(self, inputfile, line):
             if 'basis functions' in line:
                 if not hasattr(self, 'nbasis'):
                     self.set_attribute('nbasis', int(line.split()[-3]))
+                    # We can't display more MOs than there are basis
```

Comment:
By we do you mean QChem? Could you be more explicit - does QChem print more MOs than there are basis functions and we parse only a subset?